ui/src/app/views/workspace/component.ts

```diff
@@ -50,6 +51,7 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
   bugReportOpen: boolean;
   bugReportDescription = '';
   googleBucketModal = false;
+  showUpdatedResearchPurpose = environment.enableDatasetBuilder;
```

Comment:
The tying together of these features is a bit surprising – your offline explanation makes total sense, but the code here might benefit from a quick comment (and maybe a link to the dataset builder Epic) saying why we're tying these together and when it should be removed.

Reply:
will add a comment